deploy/helm/flightctl/templates/flightctl-db-secret.yaml

```
@@ -0,0 +1,35 @@
+{{- $namespaces := list .Release.Namespace }}
+{{- if .Values.global.internalNamespace }}
+{{- $namespaces = append $namespaces .Values.global.internalNamespace }}
+{{- end }}
+{{- $existingSecret := (lookup "v1" "Secret" .Release.Namespace "flightctl-db-secret") }}
+{{- $password := "" }}
+{{- $masterPassword := "" }}
+{{- $userPassword := "" }}
+{{- if $existingSecret }}
+  {{- $password = (index $existingSecret.data "password") }}
+  {{- $masterPassword = (index $existingSecret.data "masterPassword") }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)